src/actions/index.js

```diff
@@ -12,26 +12,28 @@ export function fetchSearchIndexSuccess(searchIndex) {
     type: actionTypes.FETCH_SEARCH_INDEX_SUCCESS,
     searchIndex,
   };
-}
+} 
 
 export function fetchSearchIndexFailure() {
   return {
     type: actionTypes.FETCH_SEARCH_INDEX_FAILURE,
   };
 }
 
-export const getSearchResultsRequest = searchInput => ({
-  type: actionTypes.GET_SEARCH_RESULTS_REQUEST,
-  searchInput,
-});
+export function fetchSubjectsRequest(){
+  return {
+    type: actionTypes.FETCH_SUBJECTS_REQUEST,
+  };
+}
 
-export const getSearchResultsSuccess = searchResults => ({
-  type: actionTypes.GET_SEARCH_RESULTS_SUCCESS,
-  searchResults,
-});
+export function fetchSubjectsSuccess(){
```

Comment:
Same thing here (space after parens)

Reply:
The success action creator will also need an input, since if the effect succeeded, we will be getting back an array of subjects.

Reply:
Oh I see you have a correct version of this down below.